dotcom-rendering/scripts/env/check-dotenv.ts

```diff
@@ -0,0 +1,16 @@
+import { promises as fs } from 'fs';
+import * as path from 'path'
+import { getGuardianConfiguration } from '../../src/server/lib/aws/aws-parameters';
+
+// eslint-disable-next-line @typescript-eslint/no-floating-promises
+(async () => {
+	const envVars = ['images.signature-salt'];
```

Comment:
Is this where the salt is currently held or is it this a new var?

Reply:
It's what it was previously.